relnotes/ipool.feature.md

```diff
@@ -0,0 +1,7 @@
+* `ocean.util.container.pool.model.IPool`, `ocean.util.container.pool.model.IAggregatePool`
+
+  A new abstract function `isBusy()` has been added to the API of IPool. Classes
+  wishing to inherit from `IPool` must provide an implementation for it.
```

Comment:
Yep, it'll be a breaking change for any class that inherits from `IPool`. Does that mean that this PR should then target `v3.x.x`?